### PR TITLE
Update Chinese Locale

### PR DESCRIPTION
**Pull request by lutinglt**
### Describe your changes:

Updated Chinese localization, most of the untranslated parts have been fixed.

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/zh-cn.json [hunk 1]
```diff
@@ -178,16 +178,16 @@
...
-    "compute-row-count": "Compute Row Count",
+    "compute-row-count": "计算行数",
     "condition": "条件",
     "config": "配置",
     "configuration": "配置",
-    "configure": "Configure",
+    "configure": "配置",
     "configure-a-service": "配置服务",
-    "configure-dbt-model": "配置 dbt 模型",
+    "configure-dbt-model": "配置 DBT 模型",
```
Comment (ulixius9) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/zh-cn.json:190]: this has to be in the lowercase according to the dbt brand guidelines

**Review by ulixius9 — COMMENTED** [review]
Hey @lutinglt, thanks for your contribution, leaving a small comment 🙇

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/zh-cn.json (hunk 1 above)
Comment (lutinglt) [reply openmetadata-ui/src/main/resources/ui/src/locale/languages/zh-cn.json:190]: I thought it was a capitalized acronym, but I'll change it back. 🤣

**Comment by lutinglt** [comment]
@ulixius9 DBT has been changed back to dbt.

**Review by ulixius9 — DISMISSED** [review]

**Review by ulixius9 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**